### PR TITLE
Typescript

### PR DESCRIPTION
**Pull request by signed**
Still have to look into the `docs`deployment.
Maybe some feedback if this works for you?

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -15,7 +60,11 @@ function rangeFromTo(from, to){
 }
 
 function range(startAt, size) {
-  return [...Array(size).keys()].map(i => i + startAt);
```
Comment (signed) [inline src/index.tsx:18]: Had to rewrite because typescript does not know about Array.keys()

File: src/lang/arrays.ts [hunk 2]
```diff
@@ -9,25 +9,25 @@ export function shuffle(array) {
   return array;
 }
 
-export function removeFromArray(a, v) {
+export function removeFromArray<T>(array: Array<T>, v: T): void {
```
Comment (kosta) [inline src/lang/arrays.ts:12]: Maybe return true if it was found? 😬

**Comment by kosta** [comment]
I will try this out today so we can merge. If you have any further changes (e.g. more type information etc.) please branch off this branch so the changes don't pile up in this PR

**Comment by signed** [comment]
I have another branch ready with test setup (jest/enzyme) based on this one. Should I open another pull request right now or wait till this is merged?

I think the webpack branch is obsolete after this is merged.

**Comment by kosta** [comment]
Sounds good I hope I can merge tonight

**Comment by kosta** [comment]
Thank you for doing this! 

# 💃

**Pull request merged**